src/hooks/__tests__/useDeleteMeeting.test.ts

```diff
@@ -0,0 +1,13 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import { useDeleteMeeting } from "../useDeleteMeeting";
+
+describe("useDeleteMeeting", () => {
+  test.todo("should removed the token from deskpro user store if it's an instant meeting");
+
+  test.todo("should deleted meeting from zoom api");
+
+  test.todo("should run invalidate cache");
+
+  test.todo("should handle error catch in each case");
```

Comment:
I can only approve the PR if the tests actually work

Reply:
These are not critical tests. I'll write them after implementing the main functionality in [this story](https://app.shortcut.com/deskpro/story/106868/zoom-improvements)

Reply:
done